package.json

```diff
@@ -32,25 +32,22 @@
   },
   "homepage": "https://github.com/IQSS/dataverse-client-javascript#readme",
```

Comment:
When should we rename to repo to https://github.com/IQSS/js-dataverse ?

Reply:
Let's change it when we release the next version to npm. That way we don't affect the links to the repository that currently appear at [https://www.npmjs.com/package/js-dataverse](https://www.npmjs.com/package/js-dataverse).

What do you think?

Reply:
That's fine but it would also be fine to rename it now. GitHub does a good job of redirecting. I'm resolving this but we can rename it any time, I'd say.